src/main/scala/firrtl/transforms/BlackBoxSourceHelper.scala

```diff
@@ -150,17 +159,18 @@ object BlackBoxSourceHelper {
     out.close()
   }
 
-  val fileListName = "firrtl_black_box_resource_files.f"
```

Comment:
I think it'd be good to have `fileListName` as a deprecated `def` just in case someone is using it

Reply:
Good call